javascript/ql/src/Security/CWE-020/UselessRegExpCharacterEscape.qhelp

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE qhelp PUBLIC
+"-//Semmle//qhelp//EN"
+"qhelp.dtd">
+<qhelp>
+
+	<overview>
+		<p>
+
+			Characters in string/template literals, and
+			regular expression literals can be backslash-escaped by prepending a
+			backslash to them. A common example of this is <code>"\n"</code> which
+			corresponds to a single <code>newline</code>.
+
+			However, not all characters change behavior when they are
+			backslash-escaped. In this case, the prepended backslash just makes
+			the character appear to be backslash-escaped during a cursory look at the source
+			code, even though the character is not backslash-escaped at runtime. An example
+			of this is <code>"\k"</code> which is equivalent to <code>"k"</code>.
+
+			Such superfluous backslash-escapes are usually benign, and
+			do not change the behavior of the program.
+
+		</p>
+
+		<p>
+
+			The set of characters that change meaning when they are
+			backslash-escaped is different for regular expression literals and
+			string/template literals.
```

Comment:
The `string/template literal` wording seems unnecessary, `string literal` should be fine (here and below).